packages/components/psammead-story-promo-list/src/index.jsx

```diff
@@ -12,22 +12,23 @@ import {
   GEL_GROUP_4_SCREEN_WIDTH_MIN,
 } from '@bbc/gel-foundations/breakpoints';
 
-export const StoryPromoLi = styled.li.attrs({
+export const StoryPromoLiBase = styled.li.attrs({
   role: 'listitem',
 })`
-
-  @media (max-width: ${GEL_GROUP_3_SCREEN_WIDTH_MAX}) {
-    border-bottom: 0.0625rem solid ${C_LUNAR};
-  }
-
   ${({ border }) =>
-    !border &&
+    border &&
```

Comment:
It looks like this block could be missing some styles:

If a border is set, do we need to ensure it isn't added to the last child?

Reply:
Good spot @simonsinclair, fixed.